lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -43,6 +43,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
+    <PackageReference Include="WebDriverBiDi" Version="0.0.9" />
```

Comment:
Would be convenient if `WebDriverBiDi` had a .NET target without any nuget dependencies.

Reply:
FYI: `WebDriverBiDi` has a newer release with the added TFMs
![image](https://github.com/user-attachments/assets/3bde0d87-7c1f-4f9e-ac3b-a1fb4cbfbc69)

Reply:
Yes. I'm working on a new PR that needs that version 😉